python/server/ServerWrapper.cpp

```diff
@@ -905,6 +905,25 @@ namespace {
         return field->ID();
     }
 
+    // Return all systems within \p jumps of \p sys_ids
+    list SystemsWithinJumps(size_t jumps, list sys_ids) {
+        list py_systems;
+        boost::python::stl_input_iterator<int> end;
+
+        std::vector<int> systems;
+
+        for (boost::python::stl_input_iterator<int> id(sys_ids); id != end; ++id) {
+            systems.push_back(*id);
+        }
+
+        auto near = GetPathfinder()->WithinJumps(jumps, systems);
```

Comment:
After looking into the other changes why not use 'systems_in_vicinity'?  It was already a good name and the documentation for the python `get_systems_within_jumps` just provided the information I am missing out at `WithinJumps` so it should be transferred and adapted accordingly.

> I've found that no matter what names I choose someone finds them too terse, too verbose, too cryptic or all of the above.

That's the outcome of cooperative development. Everybody has to deal with different opinions. ;)  In genreal I will always complain about terse or generalized names if the context doesn't provide any information about the code usage. In this case `near` just didn't ring a bell.